src/pages/AddBook.jsx

```diff
@@ -177,42 +178,54 @@ export default function AddBookPage() {
               </label>
             </form>
 
-            <div className="flex items-center h-16 bg-gray-50 ">
-              <button
-                className="h-10 ml-4 font-semibold text-white rounded-md bg-booklistBlue-dark w-28"
-                type="submit"
-                form="new book"
-                onClick={addBookHandler}
-              >
-                Save
-              </button>
+            <div className='flex items-center h-16 bg-gray-50 '>
+              {!title ? (
+                <h4>Choose a book to save changes.</h4>
```

Comment:
Instead of a having this text let's ghost out the save button and make it unclickable when the title, author, or list inputs are empty